scripts/install

```diff
@@ -115,10 +112,10 @@ ynh_system_user_create --username=$app
 #=================================================
 # PHP-FPM CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring php-fpm..." --weight=2
+ynh_script_progression --message="Configuring php-fpm..." --weight=50
 
 # Create a dedicated php-fpm config
-ynh_add_fpm_config --usage=medium --footprint=high
+ynh_add_fpm_config --usage=medium --footprint=high --phpversion=$YNH_PHP_VERSION --package="$extra_php_dependencies"
```

Comment:
Does this line needs to honor `$fpm_usage` `$fpm_footprint` or I'm missing something ?

Reply:
The helper `ynh_add_fpm_config` stores these variables :-)

https://github.com/YunoHost/yunohost/blob/051d14a945ef00317cf79061389fb15b0fee951b/data/helpers.d/php#L168